From ae3b846770145dd4f70359cf030bec9ab7851bcb Mon Sep 17 00:00:00 2001
From: Igor Patrik <igorp@marvell.com>
Date: Wed, 3 Jul 2013 12:44:19 +0200
Subject: [PATCH 0762/1825] axp: Fixed VMALLOC_END includes

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit f5d9cae214c6771bd8a4a64751a1653b36793204

	VMALLOC_END may be same for AXP and ALP,
	The comment about Dove LCD outdated.
	This fix was inserted as part of cleanup and wrong for AXP.

This reverts commit 86bb8c2f0020884210535e5f9dd5115554cde067.

Change-Id: If4243288adc9f418f678ab05f27008c90249818b
Signed-off-by: Igor Patrik <igorp@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/2402
Reviewed-by: Kosta Zertsekel <konszert@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/include/asm/pgtable.h |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/arch/arm/include/asm/pgtable.h b/arch/arm/include/asm/pgtable.h
index 39aef5c..bea0c61 100644
--- a/arch/arm/include/asm/pgtable.h
+++ b/arch/arm/include/asm/pgtable.h
@@ -42,12 +42,6 @@
 #define VMALLOC_START		(((unsigned long)high_memory + VMALLOC_OFFSET) & ~(VMALLOC_OFFSET-1))
 #ifdef CONFIG_ARCH_MX6Q
 #define VMALLOC_END		0xfb000000UL
-#elif defined(CONFIG_ARMADA_XP)
-/* Dove LCD driver performs big allocations for FrameBuffer memory, we need to
- * move CONSISTENT_BASE by 32MB
- */
-/* Was 0x2000000 */
-#define VMALLOC_END		0xfa800000UL
 #else
 #define VMALLOC_END		0xff000000UL
 #endif
-- 
1.7.5.4

